### ISSUE TITLE
Some songs can't be selected because of screen resolution

### ISSUE THREAD

**Issue by corrupt21** [BenVlodgi/BEE2.4#319]
For example, forwarding the cause of science can't be selected because the button to select it goes offscreen.
For example, I have a WXGA monitor. The name is so long that the button goes offscreen.
![button](https://cloud.githubusercontent.com/assets/17777059/15089772/e9492d6e-144e-11e6-90d6-28530b140d64.png)

**Comment by Konclan** [comment]
Scroll wheel or shrink the window?

**Comment by Konclan** [comment]
Also you can navigate using the arrow keeps

**Comment by corrupt21** [comment]
Yeah it works now, but the way that it puts the whole name on the side shouldn't exist.

**Comment by TeamSpen210** [comment]
Two othdr solutions: double-click on the icon, or drag the bar on the right of fhe left scroll bar to resize the right pane.

**Comment by Luke18033** [comment]
I haven't seen this happen in a while, is it still a problem for anyone else?
